cluster/gke.go

```diff
@@ -61,6 +61,7 @@ import (
 const (
 	statusRunning = "RUNNING"
 	statusDone    = "DONE"
+	statusError   = "ERROR"
```

Comment:
these are operation statuses. the cluster status can be used from `pkg/cluster.Error`